lib/web/fetch/util.js

```diff
@@ -1428,6 +1428,34 @@ function getDecodeSplit (name, list) {
   return gettingDecodingSplitting(value)
 }
 
+const textDecoder = new TextDecoder()
+
+/**
+ * @see https://encoding.spec.whatwg.org/#utf-8-decode
+ * @param {Buffer} buffer
```

Comment:
```suggestion
 * @param {Buffer} buffer
 * @returns {String}
```

Reply:
my editor is able to infer the return type. It's not too important to have jsdoc because we don't enforce or check it.